src/Session/SessionManager/index.js

```diff
@@ -172,15 +173,15 @@ class SessionManager {
 		}
 
 		/**
-		 * generating cookie string to push to cookies 
+		 * generating cookie string to push to cookies
 		 * which will be used by response object
 		 * while making response
 		 * @type {Object}
 		 */
-		const session = {key:this.constructor.sessionKey,value:sessionId}
+		const session = {key:this.constructor.sessionKey,value:sessionId, options}
```

Comment:
Line is too long.<br>'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.